webcomponents/tool/src/main/frontend/js/rubrics/sakai-rubric-grading-comment.js

```diff
@@ -111,9 +111,6 @@ export class SakaiRubricGradingComment extends RubricsElement {
       this.criterion.comments = "";
     }
 
-    if (e) {
```

Comment:
Removing this block because is not necessary to save comments when the tooltip is being closed